streamlineTools/adminCommerceActionsContent.js

```
@@ -1,8 +1,13 @@
 // STUB | ADMIN COMMERCE ACTIONS CONTENT
 
 // VARS
-let code = ''
-let filename
+// let code = ''
+if (typeof code === "undefined") {
```

Biome (1.9.4)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)